deeplearning4j-core/src/test/java/org/deeplearning4j/base/MnistFetcherTest.java

```diff
@@ -0,0 +1,17 @@
+package org.deeplearning4j.base;
+
+import org.junit.Test;
+
+/**
+ * @author Justin Long (crockpotveggies)
+ */
+public class MnistFetcherTest {
+
+    @Test
+    public void testIrisDataFetcher() throws Exception {
+        MnistFetcher mnistFetcher = new MnistFetcher();
+        mnistFetcher.downloadAndUntar();
+
```

Comment:
Add an assertion here for  the files being downloaded and extracted successfully?